features/lwipstack/mbed_lib.json

```diff
@@ -100,6 +100,10 @@
             "help": "Maximum number of retransmissions of SYN segments. Current default (used if null here) is set to 6 in opt.h",
             "value": null
         },
+        "tcp-close-timeout": {
+            "help": "Maximum number of 5ms intervals for TCP close handshaking timeout",
```

Comment:
Make the config be milliseconds - exposing 5ms interval multiple is a bit weird. Even if you have 5ms granularity, you can subtract 5 from `check_limit` the loop (but make it signed).

Or I'd be quite happy with 1ms polling anyway; I think being more responsive to close - shutting down 4ms faster - is probably more of a power saving than only checking every 5 milliseconds.